Source/WebKit/WebProcess/WebPage/VisitedLinkTableController.messages.in

```diff
@@ -22,7 +22,7 @@
 
 messages -> VisitedLinkTableController {
     SetVisitedLinkTable(WebKit::SharedMemory::Handle handle)
-    VisitedLinkStateChanged(Vector<WebCore::SharedStringHash> linkHashes)
+    VisitedLinkStateChanged(Vector<uint32_t> linkHashes)
```

Comment:
This seems like a step backwards.

Reply:
I don't completely disagree, but the C++ still has the more descriptive name.